ModulePresentationsForCAP/gap/ModulePresentationsForCAP.gi

```diff
@@ -1814,18 +1823,11 @@ InstallGlobalFunction( ADD_LIFT_AND_COLIFT_LEFT,
        
     s := NrColumns( I );
     
-    if v = 0 or s = 0 then 
-    
-       XX := HomalgZeroMatrix( v, s, homalg_ring );
-       
-       return PresentationMorphism( Range( morphism_1 ), XX, Range( morphism_2 ) );
-       
-    fi;
-    
     N := UnderlyingMatrix( Source( morphism_1 ) );
     
-    if NrColumns( N ) = 0 then return ZeroMorphism( Range( morphism_1 ), Range( morphism_2 ) ); fi;
-    
+    # NrColumns( N ) = 0 implies coker(N)=0 and  s = 0 implies coker(I)=0, hence morphism_2 is zero, and the zero morphism can always be colifted.
+    if NrColumns( N ) = 0 or s = 0 then return ZeroMorphism( Range( morphism_1 ), Range( morphism_2 ) ); fi;
```

Comment:
Since we do not require a colift to be something unique, the zero morphism should always fit here (this is not a request for changes, just a justification).